src/module-1/cutStrings/index.js

```diff
@@ -1 +1,11 @@
-export const cutStrings = (arr = []) => {};
+export const cutStrings = (arr = []) => {
+   const result = arr.map(word => {
+      return word.length
+   })
+   let minElm = Math.min(...result)
```

Comment:
If a variable is never reassigned, using the `const` declaration is better
`const minElm = Math.min(...result)`
https://dmitripavlutin.com/javascript-variables-best-practices/#1-prefer-const-otherwise-use-let